### PR TITLE
Fix help for livesync command and debug command

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by dtopuzov** [comment]
:+1:

File: docs/man_pages/project/testing/debug-android.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Debugs your project on a connected device, in a native emulator or in Genymotion
 * `--stop` - Detaches the debug tools.
 * `--get-port` - Retrieves the port on which you are debugging your application.
 * `--debug-port` - Sets a new port on which to attach the debug tools.
+* `--timeout` - Sets the number of seconds that NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 60 seconds.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/debug-android.md]: the NativeScript CLI

Comment (ikoevska) [inline docs/man_pages/project/testing/debug-android.md]: Add this option to the synopsis above

File: docs/man_pages/project/testing/livesync-ios.md [hunk 2]
```diff
@@ -3,13 +3,14 @@ livesync ios
 
 Usage | Synopsis
 ------|-------
-General | `$ tns livesync ios [--device <Device ID>] [--watch]`
+General | `$ tns livesync ios [--device <Device ID>] [--watch] [--emulator]`
 
-Synchronizes the latest changes in your project to iOS devices. 
+Synchronizes the latest changes in your project to iOS devices or iOS simulator.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: or the iOS Simulator.

File: docs/man_pages/project/testing/livesync-ios.md [hunk 3]
```diff
@@ -3,13 +3,14 @@ livesync ios
...
-General | `$ tns livesync ios [--device <Device ID>] [--watch]`
+General | `$ tns livesync ios [--device <Device ID>] [--watch] [--emulator]`
 
-Synchronizes the latest changes in your project to iOS devices. 
+Synchronizes the latest changes in your project to iOS devices or iOS simulator. 
 
 ### Options
 * `--watch` - If set, when you save changes to the project, changes are automatically synchronized to the connected device.
 * `--device` - Specifies the serial number or the index of the connected device to which you want to synchronize changes. To list all connected devices, grouped by platform, run `$ tns device`
+* `--emulator` - If set, changes are synchronized to iOS simulator.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: If set, synchronizes changes to the iOS Simulator.

File: docs/man_pages/project/testing/livesync-ios.md (hunk 2 above)
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: Run a check for this description (it should be appearing in the index.md and the Related Commands across the testing section). Update it in all instances to match the description here.

File: docs/man_pages/project/testing/livesync-ios.md [hunk 4]
```diff
@@ -3,13 +3,14 @@ livesync ios
 
 Usage | Synopsis
 ------|-------
-General | `$ tns livesync ios [--device <Device ID>] [--watch]`
+General | `$ tns livesync ios [--device <Device ID>] [--watch] [--emulator]`
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: Can you have --device and --emulator simultaneously? If not, split this into a few rows:

```
Synchronize changes to all connected devices | `$ tns livesync ios [--watch]`
Synchronize changes to a selected device | `$ tns livesync ios --device <Device ID> [--watch]`
Synchronize changes to the iOS Simulator | `$ tns livesync ios --emulator [--watch]`
```

File: docs/man_pages/project/testing/livesync-ios.md (hunk 3 above)
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: Should the simulator be running? If yes, state this.

**Pull request merged**